docs/topics/request-response.rst

```diff
@@ -119,8 +118,9 @@ Request objects
        raised while processing the request. This includes pages that failed
        with 404 HTTP errors and such. It receives a `Twisted Failure`_ instance
        as first parameter.
-       For more information,
-       see :ref:`topics-request-response-ref-errbacks` below.
+       Please note that setting an ``errback`` without an explicit ``callback``
+       will result in an error.
```

Comment:
Why is that? It looks more like a bug to me, which we should fix, not document :)

Reply:
I assumed there was a reason for it, since it's a deliberate `assert` and not an unhandled exception, but it is a rather [old line](https://github.com/scrapy/scrapy/commit/73305b1eb333d6cf2c42b0cb45e4ed75e9c523a6#diff-adcd0debfae0b9c2b1f34a01d2aafc53R30) and perhaps there is no such reason anymore.  Let me think about alternative ways to approach this.